### PR TITLE
automate device enrollment key rotation in google core iot

### LINKED ISSUE #278: Write code to automate device enrollment/key rotation in Google Core IoT.
In the "Google Core IoT" example we have the command line items for how to enroll a device. However, due to the fact that we use seven day certs, we need to streamline this process in the form of some simple python code (where the user just specifies what Device IDs to enroll).

### PR DESCRIPTION
**Pull request by eugenosm**
closes https://github.com/WoTTsecurity/api/issues/278

### CODE REVIEW

File: cert-rotate/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  After the account was created press on the `Create Key` button and save the JSON file.
+- Install requirements.
+
+```shell
+python3 -m venv ~/.gc-rt-venv
+source ~/.gc-rt-venv/bin/activate
+pip install -r requirements.txt
+```
+
+Update `PROJECT_ID`, `CLOUD_REGION`, `REGISTRY_ID`, `SERVICE_ACCOUNT_JSON` and `token` values according to your project and security settings.
```
Comment (vpetersson) [inline cert-rotate/README.md]: No, these needs to be environment variables.

File: cert-rotate/cert-rotate.py [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+import datetime
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from google.oauth2 import service_account
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+
+
+# Google IoT core project settings
+PROJECT_ID = "project-id"
```
Comment (vpetersson) [inline cert-rotate/cert-rotate.py]: This needs to be `os.getenv('PROJECT_ID')` or similar (and the same for all configs below.

Comment (eugenosm) [reply cert-rotate/cert-rotate.py]: OS Environment? Or the config file would be better?

Comment (vpetersson) [reply cert-rotate/cert-rotate.py]: Environment is basically the new standard way of doing these. In particular if you'd run this inside say a dockerized environment (which would be advised).

Comment (eugenosm) [reply cert-rotate/cert-rotate.py]: Moved to os environment variables

**Comment by vpetersson** [comment]
Thanks @eugenosm!

**Pull request merged**